### PR TITLE
[GSOC][TMVA][SOFIE] Added support for standalone  MatMul operator to be accepted by Gemm Operator

### PR DESCRIPTION
**Pull request by Neel-Shah-29**
# This Pull request: 
1) Adds the Swish Activation function in Keras Parser
2) Standalone Matmul support for Gemm operator

## Checklist:

- [x] tested changes locally
- [ ] updated the docs (if necessary)

This PR fixes #

### CODE REVIEW

File: tmva/pymva/src/RModelParser_Keras.cxx [hunk 1]
```diff
@@ -71,6 +72,7 @@ const KerasMethodMap mapKerasLayer = {
    {"BatchNormalization", &MakeKerasBatchNorm},
    {"Reshape", &MakeKerasReshape},
    {"Concatenate", &MakeKerasConcat},
+   {"Swish", &MakeKerasSwish},
```
Comment (sanjibansg) [inline tmva/pymva/src/RModelParser_Keras.cxx:75]: nit: I believe the activation name internally is `swish`, and this will be case-sensitive.

**Review by sanjibansg — CHANGES_REQUESTED** [review]

File: tmva/sofie_parsers/src/ParseMatMul.cxx [hunk 2]
```diff
@@ -0,0 +1,49 @@
+#include "TMVA/RModelParser_ONNX.hxx"
```
[the remark this answers is not in the document]
Comment (Neel-Shah-29) [reply tmva/sofie_parsers/src/ParseMatMul.cxx:1]: Ok sure. I will create a separate PR for it

File: tmva/pymva/src/RModelParser_Keras.cxx (hunk 1 above)
Comment (Neel-Shah-29) [reply tmva/pymva/src/RModelParser_Keras.cxx:75]: Thanks for pointing out. I will make the changes.

File: tmva/pymva/test/generateKerasModels.py [hunk 3]
```diff
@@ -141,7 +141,20 @@ def generateActivationModel():
     model.compile(loss='mean_squared_error', optimizer=SGD(learning_rate=0.01))
     model.fit(x_train, y_train, epochs=10, batch_size=1)
     model.save('KerasModelActivations.h5')
-    
+
+def generateSwishModel():
+# Create the Keras model
+    model = Sequential()
+    model.add(Dense(64, activation='swish', input_shape=(8,)))
```
Comment (sanjibansg) [inline tmva/pymva/test/generateKerasModels.py:148]: We might have to check for the version when `swish` was introduced in Keras, and restrict the tests accordingly.

**Review by sanjibansg — APPROVED** [review]
LGTM!
Thanks for the contribution.

**Review by lmoneta — APPROVED** [review]
LGTM

**Pull request merged**